server/models/video/video-format-utils.ts

```diff
@@ -193,7 +194,9 @@ function videoFilesModelToFormattedJSON (
         torrentUrl: model.getTorrentUrl(videoFile, baseUrlHttp),
         torrentDownloadUrl: model.getTorrentDownloadUrl(videoFile, baseUrlHttp),
         fileUrl: model.getVideoFileUrl(videoFile, baseUrlHttp),
-        fileDownloadUrl: model.getVideoFileDownloadUrl(videoFile, baseUrlHttp)
+        fileDownloadUrl: model.getVideoFileDownloadUrl(videoFile, baseUrlHttp),
+        metadata: videoFile.metadata,
```

Comment:
I don't think we should include metadata in this JSON. Clients should always fetch the `metadaUrl` to get these information.

Reply:
Why? Updates will refresh the metadata anyway.

Reply:
Let the remote instance decide (some of them may not be interested in metadata).